src/futil.js

```diff
@@ -1,41 +0,0 @@
-import _ from 'lodash/fp'
-import F from 'futil'
-
-export let flattenProp = _.curry((prop, target) =>
-  _.flow(F.expandObject(_.get(prop)), _.unset(prop))(target)
-)
-
-// See R.evolve, but supports lodash nested paths
-// NOTE: does not run on objects where path is missing. flow with defaults/ensureKeys if desired
-// CAVEAT: When using `_.curry`, fast-csv's transform doesn't work
-export let updateMany = rules => data => {
-  let clone = _.cloneDeep(data)
-  F.eachIndexed((display, field) => {
-    if (_.has(field, clone))
-      F.updateOn(field, value => display(value, clone), clone)
-  })(F.compactObject(rules))
-  return clone
-}
-
-// F.arrayToObject with keys as array values
-// (['a', 'b'], x => x + 'c') => { a: 'ac', b: 'bc' }
-export let keysToObject = _.curry(
```

Comment:
Same here - but if they're not needed, they shouldn't be here.

They are pretty useful though, we should move them to futil